grpc/protos/registry/ingest/ingest.proto

```diff
@@ -17,6 +17,10 @@ message IngestRequest {
     string har = 1;
     // path_hint is a hint to the ingest service about the structure of the request path
     string path_hint = 2;
+    // api_id is used to associate requests with a particular Api in the Speakeasy platform
```

Comment:
should we make these optional so we don't enforce this across all SDKs ?

Reply:
so with protobuf 3 they are optional by default and just get filled with a default falsey value if not set